terra-notebooks-playground/Hail - How to run a notebook in the background.ipynb

```diff
@@ -0,0 +1,224 @@
+    "\n",
+    "<div class=\"alert alert-block alert-success\">\n",
+    "    <b>If you wish to capture all Hail notebook cell outputs</b>, use this notebook to run your long-running Hail notebook (or any other long-running notebook).<p>But also note that your analysis <a href=\"https://support.terra.bio/hc/en-us/articles/360029761352-Preventing-runaway-costs-with-notebook-auto-pause-#h_de5698f5-3c82-4763-aaaf-ea7df6a1869c\">must complete within 24 hours</a>.</p>\n",
+    "</div>\n",
+    "\n",
+    "How to use this notebook:\n",
+    "1. Copy this notebook to the workspace that contains the long-running notebook you wish to run in the background.\n",
+    "1. Edit the filename in `NOTEBOOK_TO_RUN` to be the name of the notebook in this workspace you wish to run.\n",
+    "1. Use menu `Cell -> Run All` to tell the kernel to run this notebook to completion.\n",
+    "1. Close this tab and do other work. (note: when you close the tab, the outputs in *this* notebook will no longer be updated, and that is fine, because what we really want are the outputs in the Hail notebook.)\n",
+    "1. When the Hail job is complete:\n",
+    "    1. Your Hail cluster will pause (if you have no other notebooks open).\n",
+    "    1. You will see a copy of your notebook in the notebooks tab with a date and timestamp suffix --> this is the output file.\n",
+    "    1. You will also see the Hail log in `gs://<workspace bucket>/hail-logs/YYYYMMDD/hail*.log`\n",
+    "\n",
+    "If you check back, and you do not see the output notebook, the Hail job is still running. To confirm this, first notice whether your Cloud analysis environment is still running. If it is, from the terminal run `yarn application -list` to see that a spark context is still processing data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import nbformat\n",
+    "from nbconvert.preprocessors import CellExecutionError\n",
+    "from nbconvert.preprocessors import ExecutePreprocessor\n",
+    "\n",
+    "import os\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "<b>Change the cell below</b> to hold the name of the notebook in this workspace that you wish to run in the background. There is no need to modify any of the other cells.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#---------[ CHANGE THIS TO BE THE NAME OF THE NOTEBOOK YOU WISH TO RUN ]-----------\n",
```

Comment:
Not necessary but you could use a display box for users here 

```
# Create and display input box
from IPython.display import display, clear_output
from ipywidgets import widgets
input_text = None
input_text = widgets.Text()
print("Enter the name of the notebook in the box below and press Enter/Return.")
display(input_text)

def handle_submit(sender):
    print(f"Notebook set to: {input_text.value}")
    
input_text.on_submit(handle_submit)
NOTEBOOK_TO_RUN = input_text.value
```

Reply:
thanks @bshifaw ! I'm going to keep it simple for now and not include any UI elements.